src/configs/winston.ts

```diff
@@ -0,0 +1,43 @@
+import * as winston from 'winston';
+import os from 'os';
+
+const homedir = os.homedir();
+const logsPath = `${homedir}/.express/logs/reveal-express-server.log`
+
+// define the custom settings for each transport (file, console)
+const options = {
+  file: {
+    level: 'info',
+    filename: logsPath,
+    handleExceptions: true,
+    json: true,
+    maxsize: 5242880, // 5MB
+    maxFiles: 5,
+    colorize: false,
+    format: winston.format.simple(),
+  },
+  console: {
+    level: 'debug',
+    handleExceptions: true,
+    json: false,
+    colorize: true,
```

Comment:
Do we need to make any of these configurable?

Reply:
addressed